tests/preprocessing/test_positionfixes.py

```diff
@@ -228,9 +237,31 @@ def test_temporal(self):
 class TestGenerate_triplegs:
     """Tests for generate_triplegs() method."""
 
-    def test_invalid_isolates(self, isolated_positionfixes):
+    def test_noncontinuous_unordered_index(self, example_positionfixes_isolated):
+        """The unordered and noncontinuous index of pfs shall not affect the generate_triplegs() result."""
```

Comment:
I think it is "unordered" but "non-continuous"

Reply:
I am not an expert in English, but in the dictionary it says [noncontinuous](https://www.collinsdictionary.com/dictionary/english/noncontinuous)